api/relationships.go

```diff
@@ -145,6 +182,17 @@ func (entity *RelationshipsCohabitants) Valid() (bool, error) {
 	return entity.CohabitantList.Valid()
 }
 
+// ClearNos clears any questions answered nos on a kickback
+func (entity *RelationshipsCohabitants) ClearNos() error {
+
+	entity.HasCohabitant.ClearNo()
+
+	entity.CohabitantList.ClearBranchNo()
```

Comment:
way to ruin my abstraction proposal, cohabitation ;_;

though theoretically this one could override from the base